app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
+    @post = Post.new(post_params)
+ 
+    if @post.save
+      redirect_to @post
+        flash[:notice] = 'Your post was succefully created'
```

Comment:
Flash needs to come be4 the `redirect_to`

Reply:
Please mind the code formatting

Reply:
done